code/parse/parselo.cpp

```diff
@@ -1391,17 +1394,67 @@ void stuff_string_line(SCP_string &outstr)
 // the default string length if using the F_NAME case.
 char *stuff_and_malloc_string(int type, const char *terminators)
 {
-	SCP_string tmp_result;
-
+	SCP_string tmp_result;  
 	stuff_string(tmp_result, type, terminators);
-	drop_white_space(tmp_result);
-
+	tmp_result=message_translate_tokens(tmp_result.c_str()); 
```

Comment:
So, the problem with this solution is that adding `message_translate_tokens` to this function will change the behavior everywhere it's called. It is called in at least 7 other places and that could have unintended effects.  

`mission_branch_brief_sound` is set using this function, for instance, and I believe that may cause the sound not to play in edge cases.  There should be a way to ensure that `message_translate_tokens` is only called when intended.